### PR TITLE
Ht 71 track auth status in local storage

### PR DESCRIPTION
**Pull request by kalibria**
Created class AuthenticationManager whose methods check the response of request and assign true or false to Local Storage.

### CODE REVIEW

**Comment by ikushlianski** [comment]
You should point such PRs that contain other PRs not to `main` but to the branch of that other PR. Thus if PR#1 is merged, Github would automatically re-point your PR to `main`

**Comment by ikushlianski** [comment]
This is required because now your PR2 contains changes from 17 files, while actually you changed just a couple of files in your PR2. If you point PR2 into PR1 then only that clear file diff will be visible

File: src/auth/authenticationManager.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      isSuccess: true,
+    };
+  }
+
+  private static getErrorCode(result: UseQueryStateResult<any, any>): number {
+    return result?.error?.status;
+  }
+
+  private setIsAuthenticated() {
+    localStorage.setItem('sessionID', 'true');
```
Comment (ikushlianski) [inline src/auth/authenticationManager.ts]: 'sessionID' should be something like `isAuthenticated`

Comment (kalibria) [reply src/auth/authenticationManager.ts]: renamed

**Pull request merged**